### DISCUSSION TITLE
v2: Adding additional markdown processor / UML Diagrams

### DISCUSSION THREAD

**Discussion by jasheal** [facebook/docusaurus#4348] [Q&A]
We would like to add markdown UML Diagrams lie graphviz, flow, gantt etc on our docs site [veramo.io](https://veramo.io)

Something like this would be awesome: https://hackmd.io/MathJax-and-UML?both

Is this currently possible and if so how would I go about it? I can't see any reference in plugins to modify the mdx processor but perhaps I've missed something.

**Comment by slorber** [comment]
Hi,

There was a recent discussion regarding MathJax recently with @guillaume-ion here https://github.com/facebook/docusaurus/discussions/4333

Maybe you should get in touch and figure out?

Some also integrated MermaidJS: https://github.com/facebook/docusaurus/issues/1258

See also how to use Latex maths: https://docs.theochu.com/docusaurus/latex/

And please add your site to. our showcase :)
https://v2.docusaurus.io/showcase

**Reply by jasheal** [reply]
Thanks for this - will do :)

**Comment by guillaume-ion** [comment]
Hey, I'm not sure if it's relevant to your use case. But I've worked on something similar.
Not directly with Graphviz but with Plantuml. 

They have a js library that you can add to your project and directly input plantuml code in tags, 
It will generate and display the corresponding diagram.

https://plantuml.com/fr/jquery

When it comes to mathjax this thread has been really helpful : https://github.com/facebook/docusaurus/issues/2078
